x-pack/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/advanced_section/advanced_section.tsx

```diff
@@ -71,13 +82,39 @@ export const AdvancedSection: FC<Props> = ({ advancedExpanded, setAdvancedExpand
         >
           <EuiFlexItem>
             <ModelPlotSwitch />
-            <AnnotationsSwitch />
-            <ModelMemoryLimitInput />
           </EuiFlexItem>
           <EuiFlexItem>
             <DedicatedIndexSwitch />
           </EuiFlexItem>
         </EuiFlexGroup>
+
```

Comment:
a bit of churn here to rearrange how the sections are laid out to get the spacing between them correct.